runtime/doc/todo.txt

```diff
@@ -546,11 +546,6 @@ is confusing.  Should say "the argument list is empty".
 xterm supports escape sequences to mark a paste operation.  Need to be
 enabled. (Bruno Sutic, 2014 Jul 11)  How to know the terminal supports this?
 
-URXVT:
-- will get stuck if byte sequence does not contain the expected semicolon.
-- Use urxvt mouse support also in xterm.  Explanations:
-  http://www.midnight-commander.org/ticket/2662
```

Comment:
@tarruda Does libtermkey or whatever report mouse coordinates using the "new way" described at http://www.midnight-commander.org/ticket/2662 ? (It's a well-written ticket worth reading, could save us time)